File: Javascript/1010.js
var input = `12 1 5.30
16 2 5.10`;
var lines = input.split(/[\s]/);
console.log(
  "VALOR A PAGAR: R$ " + (lines[1] * lines[2] + lines[4] * lines[5]).toFixed(2)
);

File: Kotlin/1010.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    input.next();
    val res = input.nextDouble() * input.nextDouble()
    input.next();
    val res2 = res + input.nextDouble() * input.nextDouble()
    println("VALOR A PAGAR: R$ %.2f".format(res2));
}